scripts/build/arch/alpha.sh

```diff
@@ -4,3 +4,16 @@ CT_DoArchTupleValues () {
     # The architecture part of the tuple:
```

Comment:
It helps for bisecting and general "trying to find out where things changed". If I see (e)glibc-common I start thinking of older code that isn't relative anymore. Yes, it's a bit of nitpicking, but theres a method to the madness.